tests/lib/linter.js

```diff
@@ -60,6 +59,14 @@ function getVariable(scope, name) {
     return scope.variables.find(v => v.name === name) || null;
 }
 
+/**
+ * `eslint-env` comments are processed by doing a full source text match before parsing.
+ * As a result, if this source file contains `eslint- env` followed by an environment in a string,
+ * it will actually enable the given envs for this source file. This helper function is used to avoid having a string
```

Comment:
This line still refers to this as a function. I'm on mobile so I can't (reliably) generate a suggestion, sorry.

Reply:
Oops, thanks for pointing that out.